docs/Development Concepts/4.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Transports in Waku
+---
+
+Waku v2 is built in top of libp2p, and like libp2p it strives to be transport agnostic. We define a set of recommended transports in order to achieve a baseline of interoperability between clients.
+
+This section describes these recommended transports:
+
+- Waku client implementations SHOULD support the TCP transport.
+
+- Where TCP is supported it MUST be enabled for both dialing and listening, even if other transports are available.
```

Comment:
Hum,. where did you get that? RFC? Unfortunately it's not always practical to listen.